src/libPOW/pow.h

```diff
@@ -102,7 +102,11 @@ class POW
     ethash_light_t ethash_light_client;
     uint64_t currentBlockNum;
     std::atomic<bool> m_shouldMine;
-    std::unique_ptr<dev::eth::Miner> m_miner;
+    std::vector<dev::eth::MinerPtr> m_miners;
+    std::vector<ethash_mining_result_t> m_vecMiningResult;
+    std::atomic<int> m_minerIndex;
+    std::condition_variable m_cvMineResult;
```

Comment:
perhaps we can be consistent on the namings? Using the word `MiningResult` is better.

Reply:
Ok, I will update it.